### PR TITLE
Feature/how to play

### PR DESCRIPTION
**Pull request by JustD1d1t**
Moved variables to global file called variables.scss
Moved global styles to file called styles.scss
Changed webpack and add there howtoplay page

### CODE REVIEW

File: src/howtoplay/howtoplay.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          boards: yours and AI’s. Ships can be placed horizontally or vertically
+          and cannot overlap. They have different sizes: 1 of each 5-, 4-, 3-
+          square ship and 2 of each 2-, 1- square ships. The shot is made by
+          clicking on the desired square on the opponent's board. For each
+          successful hit on an AI’s ship, you get 1 point and an additional
+          point for sinking the ship. The best scores are recorded in high
+          scores. You can change the name displayed in the high scores by
+          entering your name and saving it in the Settings before the start of
+          the game, otherwise the score will be saved under the default name
+          Player.<br />Good luck, sailor!
```
Comment (ruljin) [inline src/howtoplay/howtoplay.html:27]: Don't use `<br />` in `<p>` tags.

https://developer.mozilla.org/pl/docs/Web/HTML/Element/p

File: src/styles.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```
Comment (ruljin) [inline src/styles.scss:1]: I think this file is some sort of artifact. Remove it, please. It doesn't look like it is needed anymore.

**Review by ruljin — CHANGES_REQUESTED** [review]

File: src/howtoplay/howtoplay.html (hunk 1 above)
Comment (JustD1d1t) [reply src/howtoplay/howtoplay.html:27]: moved `<p>` tags (removed br and created two tags) to `<div>` tag with class container__instruction

File: src/styles.scss (hunk 2 above)
Comment (JustD1d1t) [reply src/styles.scss:1]: Yeah. It's imported in two files. Removed

**Review by ruljin — APPROVED** [review]

**Pull request merged**